backend/.env.development

```
@@ -14,5 +14,5 @@ INSEE_HOST=https://insee.fr
 INSEE_CONSUMER_KEY=INSEE_CONSUMER_KEY
 INSEE_CONSUMER_SECRET=INSEE_CONSUMER_SECRET
 FORCE_COOKIES_SAME_SITE_PROTECTION=False
-DATAPASS_OAUTH_CLIENT_ID=https://github.com/betagouv/moncomptepro/blob/master/scripts/fixtures.sql#L237
+DATAPASS_OAUTH_CLIENT_ID=https://github.com/betagouv/moncomptepro/blob/03262ee424da5fda4c96efddf77698ba78e89751/scripts/fixtures.sql#L242
```

Et pourquoi pas une phrase derrière un commentaire, éventuellement ?
```suggestion
DATAPASS_OAUTH_CLIENT_ID= # look into https://github.com/betagouv/moncomptepro/blob/03262ee424da5fda4c96efddf77698ba78e89751/scripts/fixtures.sql#L242
```

---

1. Je ne suis pas sûr que cela soit valide dans un `.env`
2. Le commit possède à mon sens assez d'info (et de manière général message de commit >>> commentaire)

Si 1. est faux, je n'ai cependant aucun souci à appliquer la suggestion (pour moi ça m'est égal je ne lis aucun commentaire)